article/app/controllers/PublicationController.scala

```diff
@@ -22,7 +23,7 @@ class PublicationController(
     with Dates
     with GuLogging {
 
-  private val dateFormatUTC = DateTimeFormat.forPattern("yyyy/MMM/dd").withZone(DateTimeZone.UTC)
+  private val dateFormatUTC = DateTimeFormat.forPattern("yyyy/MMM/dd").withZone(DateTimeZone.UTC).withLocale(Locale.US)
```

Comment:
[September's short form "Sep" no longer parses in Java 17 in en_GB locale](https://stackoverflow.com/questions/69267710/septembers-short-form-sep-no-longer-parses-in-java-17-in-en-gb-locale) so [this test](https://github.com/guardian/frontend/blob/main/article/test/PublicationControllerTest.scala#L79-L85) was failing. We had to change locale to US for `DateTimeFormat` to successfully parse "sep".